src/main/java/com/intuit/graphql/orchestrator/common/FieldPosition.java

```diff
@@ -0,0 +1,34 @@
+package com.intuit.graphql.orchestrator.common;
+
+import com.intuit.graphql.orchestrator.authorization.FieldPath;
+import java.util.Objects;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+@Getter
+public class FieldPosition {
+
+    @EqualsAndHashCode.Include
+    private final String parentTypename;
```

Comment:
You can use graphql-javas FieldCoordinate class here for parent and fieldname

Reply:
yup this is done

Reply:
this file is removed